docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+# Create a channel without a system channel
+
+To simplify the channel creation process and enhance the privacy and scalability of channels, it is now possible to create application channels (where transactions involving assets happen) without first creating a "system channel" managed by the ordering service. Use this tutorial to learn how to create new channels without a system channel by using the [configtxgen](../commands/configtxgen.html) tool to create a genesis block and the [osnadmin CLI](../commands/osnadminchannel.html) (which runs against a REST API exposed by each ordering service node) to join ordering nodes to a channel. This process allows ordering nodes to to join (or leave) any number of channels as needed, similar to how peers can participate in multiple channels.
+
+**How this process differs from the legacy process:**
+
+ * **System channel no longer required**: Besides the creation of the system representing an extra step (as compared to the new process), this system channel created an extra layer of administration that, for some use cases, provided no tangible benefit.
+ * **Consortium no longer required**: You no longer need to define the set of organizations, known as the "consortium", who are permitted to create channels on a particular ordering service. With this new process, **all channels are application channels**, so the concept of a list of organizations who can create channels no longer applies. Any set of organizations can get together and create a channel using a defined set of ordering nodes (which become the ordering service of that channel).
+ * **Simplified ordering node creation process**: Because a genesis block no longer needs to exist before an ordering node is created, admins can now focus on the process of setting up their infrastructure and making sure their node is working correctly before joining a particular application channel. Additionally, because an ordering node no longer needs to scan the system channel ledger to determine which channels it is joined to and then replicate those ledgers, the node starts up more rapidly.
```

Comment:
> Additionally, because an ordering node no longer needs to scan the system channel ledger to determine which channels it is joined to and then replicate those ledgers, the node starts up more rapidly.

This isn't really true.  The only time the orderer scans the system channel for channels is when bootstrapping from a late join config block.  Otherwise, the existing ledgers are simply loaded.  Where did this sentence/claim originate from?

Reply:
The idea that orderers do not have to scan the ordering service for the channels anymore (and that this is an improvement over the old way) it is in the RFC under "scalability problems". It was my inference that this would help nodes start faster. 

Should this whole sentence be deleted?

Reply:
Given he said it not really true. I removed it.